blog/_posts/2020-1-23-crash-course-kubernetes-nodes.md

```diff
@@ -0,0 +1,168 @@
+---
+slug: crash-course-kubernetes-nodes
+date: 2020-1-23
+author: maia
+layout: blog
+title: "A Crash Course in Kubernetes #2: Nodes"
+image: "featuredImage.png"
+image_needs_slug: true
```

Comment:
please don't use `image_needs_slug` on new blog posts. It breaks some stuff but is necessary for backwards compatibility. Take a look at the more recent blog posts for how to do this properly

Reply:
👍